.github/workflows/test.yml

```diff
@@ -28,6 +28,9 @@ jobs:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
           CI: true
 
+      - name: A11y Testing ❤️
+        run: npx lerna run axe --stream
```

Comment:
Doesn't seem to finish the storybook build, which is weird: https://github.com/chanzuckerberg/lp-design-system/pull/167/checks?check_run_id=1120208796